certification/internal/engine/engine.go

```diff
@@ -585,3 +587,15 @@ func convertLabels(imageLabels map[string]string) []pyxis.Label {
 
 	return pyxisLabels
 }
+
+// retryOnceAfter is a crane option that retries once after t duration.
+func retryOnceAfter(t time.Duration) crane.Option {
+	return func(o *crane.Options) {
+		o.Remote = append(o.Remote, remote.WithRetryBackoff(remote.Backoff{
+			Duration: t,
+			Factor:   1.0,
+			Jitter:   0.1,
+			Steps:    2,
+		}))
```

Comment:
Hi @komish, looks good for me, you change "0s -> 1s -> 4s" [default strategy](https://github.com/google/go-containerregistry/blob/e3b94c7e7a5720c5e2985012f617f36006abff62/pkg/v1/remote/options.go#L69-L75) to "0s -> 5s", that could improve the situation in case of longer outages.

Maybe we could even keep the exponential strategy if the retry interval is large enough, something like "0s -> 5s -> 15s" strategy.

```
remote.Backoff{
			Duration: 5,
			Factor:   2.0,
			Jitter:   0.1,
			Steps:    3,
		}
```

Reply:
For the moment, I'll merge with 0s --> 5s, and if this proves not useful enough, we may consider a 3-step exponential. Keep us posted.